Python/bytecodes.c

```diff
@@ -1450,7 +1450,7 @@ dummy_func(
         op(_STORE_ATTR, (v, owner --)) {
             PyObject *name = GETITEM(FRAME_CO_NAMES, oparg);
             int err = PyObject_SetAttr(PyStackRef_AsPyObjectBorrow(owner),
-                                       name, PyStackRef_AsPyObjectSteal(v));
+                                       name, PyStackRef_AsPyObjectBorrow(v));
```

Comment:
PyObject_SetAttr actually takes borrowed refs too, so this should be stolen. This  was caught when the `DECREF_INPUTS` tried to close an already stolen stackref.